### PR TITLE
[WIP] Added start script to run the app in the emulator

### PR DESCRIPTION
**Pull request by alex-deloitte**
## Ticket title
One line description
[link to ticket number]()

## Checklist
- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: .secignore [hunk 1]
```diff
@@ -0,0 +1 @@
+env
```
Comment (alex-deloitte) [inline .secignore:1]: why do you need to add this folder to secignore?

File: README.md [hunk 2]
```diff
@@ -32,20 +34,41 @@ node_modules/.bin/ionic cordova platform remove ios
...
+You will first need to get your iPhone 8 Plus that you can find under `Simulator > Hardware menu > Devices > Manage devices`.
+The ID will be shown on the `Identifier` key.
+You can then copy paste the value in the `env` file to the following variable:
+```
+IPHONE8_P_ID=
+
+```
+
+
+and rename this file from `env` to `.env`.
```
Comment (alex-deloitte) [inline README.md:68]: Why did you add this additional step?

Comment (JonLazariniDD) [reply README.md:106]: `env` file is a placeholder file for now that will store any env variables to be injected at build time.
`.env` file is sourced before the script so the environment variables become available within the shell and we only get to do that once and no longer need to manually repeat these tasks over again.
`env` will become `.env` with the relevant environment variables added.
I was thinking that having to rename a file scaffolded rather than letting the user creates it could reduce potential errors when setting everything up.
